week-2/extended-project/yoda-y-sus-apis/web_api_methods.rb

```diff
@@ -0,0 +1,36 @@
+require "http"
+require "json"
+require "csv"
+
+$index_general = 0
+$uri = "https://www.cnet.com/es/noticias/ranking-peliculas-star-wars-guerra-de-las-galaxias/"
+$response = ""
+def initialization_scrapping
+  $response = HTTP.get($uri).body.to_s
+  text_references = "Ránking de las películas de 'Star Wars' según CNET en Español"
+  $index_general = /#{text_references}/ =~ $response
+end
+
+def get_score(name_movie)
```

Comment:
I prefer names without the `get_` prefix, just `score` (this applies to the other methods too!)